barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_to_constraint_buf.cpp

```diff
@@ -307,14 +336,23 @@ void handle_blackbox_func_call(Program::Opcode::BlackBoxFuncCall const& arg,
                     .num_bits = arg.input.num_bits,
                 });
                 af.original_opcode_indices.range_constraints.push_back(opcode_index);
-
+                if (af.minimal_range.contains(witness_input)) {
```

Comment:
0 is not a good default value for the minimal range, I'd prefer not having this value in the minimal_range map.

Reply:
Oh no, 0 would never be added since the only cases here are a) the range has already been set or b) the range has not been set (and thus gets default initialized to 0) then immediately gets set to `arg.input.num_bits`. The behavior is identical to what you have here. Fine to leave it as is though